zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/scheduler/QuartzSchedulerService.java

```diff
@@ -54,6 +54,19 @@ public QuartzSchedulerService(ZeppelinConfiguration zeppelinConfiguration, Noteb
     this.notebook = notebook;
     this.scheduler = new StdSchedulerFactory().getScheduler();
     this.scheduler.start();
+
+    Thread loadingNotesThread = new Thread(){
```

Comment:
Wouldn't be easier to write it with lambda syntax `() -> ...` ?

Reply:
Thanks for the review @alexott this comment is addressed